docs/releases/v1.23.0.md

```diff
@@ -0,0 +1,79 @@
+---
+id: v1.23.0
+title: v1.23.0
+description: Backstage Release v1.23.0
+---
+
+These are the release notes for the v1.23.0 release of [Backstage](https://backstage.io/).
+
+A huge thanks to the whole team of maintainers and contributors as well as the amazing Backstage Community for the hard work in getting this release developed and done.
+
+## Highlights
+
+### Auth module migrations
+
+Several more auth providers have been migrated to be implemented as standalone modules, adding support for them in the [new backend system](https://backstage.io/docs/backend-system/building-backends/migrating#the-auth-plugin). The migrated providers are `microsoft`, `oidc`, and `aws-alb`. If you detect any issues, please reach out on [Discord](https://discord.gg/backstage-687207715902193673) or [open an issue](https://github.com/backstage/backstage/issues).
+
+### Scaffolder
+
+We’ve removed the last of the legacy exports from `/alpha`. Please [open an issue](https://github.com/backstage/backstage/issues) if there’s a reason why you still need to use them.
+
+If you’re using the New Backend System and the Scaffolder plugin, the built in modules have been trimmed down, so you’ll need to install other modules in order to pull in additional actions. You can read more [in the migration guide](https://backstage.io/docs/backend-system/building-backends/migrating#the-scaffolder-plugin)
+
+### New Module: `@backstage/plugin-scaffolder-backend-module-gitea`
+
+Provides actions to the Scaffolder for creating repositories and pushing to Gitea. Thanks to [@cmoulliard](https://github.com/cmoulliard)!
+
+### New Modules: `@backstage/plugin-scaffolder-backend-module-bitbucket-cloud` and `@backstage/plugin-scaffolder-backend-module-bitbucket-server`
+
+Thanks to [@pjungermann](https://github.com/pjungermann)’s work, we’ve now broken apart the BitBucket module into two separate modules to provide different actions to support the different BitBucket offerings.
+
+This change also deprecates the original `@backstage/plugin-scaffolder-backend-module-bitbucket`, so if you’re using the new Backend System we encourage you to move towards using these new modules instead.
+
+### New lint rule to forbid top-level MUI imports
+
+The `@backstage/eslint-rules` package now provides the `@backstage/no-top-level-material-ui-4-imports` rule that forbids imports such as `import { Box } from '@material-ui/core'`. This rule is currently disabled by default and needs to be manually enabled in `.eslintrc.js`:
+
+```json
+module.exports = require('@backstage/cli/config/eslint-factory')(__dirname, {
+  rules: {
+    '@backstage/no-top-level-material-ui-4-imports': 'error',
+  },
+});
+
+The `backstage-cli repo fix` command will now update the `repository` fields in all workspace packages, as long as the root `package.json` has the `repository` field set.
```

Comment:
Does this need a title?

Reply:
ah quite right, the diff was misleading above